.github/workflows/hermes.yaml

```diff
@@ -76,6 +76,7 @@ jobs:
             ..#cargo-nextest \
             -c \
             cargo nextest run --release \
+            -p hermes-starknet-integration-tests \
```

Comment:
we don't have any (unit) tests at any other crates?

Reply:
Good point. I have now ignore it in 8f21ecbb66bb86231c4feb453b69877b7d5daad5.